src/layers/core/choropleth-layer/choropleth-layer.js

```diff
@@ -58,10 +67,14 @@ export default class ChoroplethLayer extends BaseLayer {
         {size: 3, update: this.calculatePickingColors, noAlloc: true}
     });
 
+    const indexType = gl.getExtension('OES_element_index_uint') ?
```

Comment:
Should we call it `IndexType` (capitalized) since it holds a type and you will call `new IndexType`?